python/ray/log_monitor.py

```diff
@@ -125,9 +125,12 @@ def update_log_filenames(self):
         log_file_paths = glob.glob(f"{self.logs_dir}/worker*[.out|.err]")
         # segfaults and other serious errors are logged here
         raylet_err_paths = glob.glob(f"{self.logs_dir}/raylet*.err")
+        # monitor event logs
```

Comment:
If I read the next line I understand it is monitoring some logs, can you make this comment more useful?